### ISSUE TITLE
Learning rate scheduler

### ISSUE THREAD

**Issue by Saquib764** [replicate/flux-fine-tuner#59]
Hi,
I have noticed that after a point during training loss doesn't change too much.

I tried using a lower learning rate to reach an even lower loss but it takes too much time.

I was wondering if we can have an option to pass a LR scheduler.

Similar to this- https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/2670
